src/main/webapp/app/home/home.component.scss

```diff
@@ -30,56 +30,59 @@ Main page styles
   margin-top: 15%;
 }
 
-#mission {
-  background-color: white;
-  padding: 1rem 0 1rem 0;
-}
+// #mission {
```

Comment:
You don't need to comment them out; just delete them if they are unnecessary. :smile: